src/Klarna/Rest/Payments/Orders.php

```diff
@@ -0,0 +1,138 @@
+
+use GuzzleHttp\Exception\RequestException;
+use Klarna\Exceptions\NotApplicableException;
+use Klarna\Rest\Resource;
+use Klarna\Rest\Transport\Connector;
+use Klarna\Rest\Transport\Exception\ConnectorException;
+
+/**
+ * Payments order resource.
+ *
+ * @example docs/examples/PaymentsAPI/Orders/cancel_existing_authorization.php Cancel an existing authorization
+ * @example docs/examples/PaymentsAPI/Orders/create_order.php Create a new order
+ * @example docs/examples/PaymentsAPI/Orders/generate_customer_token.php Generate a consumer token
+ */
+class Orders extends Resource
+{
+    /**
+     * {@inheritDoc}
+     */
+    const ID_FIELD = 'authorization_token';
+
+    /**
+     * {@inheritDoc}
+     */
+    public static $path = '/payments/v1/authorizations';
+
+    /**
+     * Constructs an order instance.
+     *
+     * @param Connector $connector HTTP transport connector
+     * @param string    $authorizationToken   Authorization Token
+     */
+    public function __construct(Connector $connector, $authorizationToken)
+    {
+        parent::__construct($connector);
+
+        $this->setLocation(self::$path . "/{$authorizationToken}");
+        $this[static::ID_FIELD] = $authorizationToken;
+    }
+
+    /**
+     * Not applicable.
+     *
+     * @throws NotApplicableException
+     */
+    public function fetch()
+    {
+        throw new NotApplicableException('Not applicable');
+    }
+
+    /**
+     * Creates the resource.
+     *
+     * @param array $data Creation data
+     *
+     * @throws ConnectorException When the API replies with an error response
+     * @throws RequestException   When an error is encountered
+     * @throws \RuntimeException  If the location header is missing
+     * @throws \RuntimeException  If the API replies with an unexpected response
+     * @throws \LogicException    When Guzzle cannot populate the response
+     *
+     * @return array Order data
+     */
+    public function create(array $data)
+    {
+        return $this->post($this->getLocation() . '/order', $data)
+            ->status('200')
+            ->contentType('application/json')
+            ->getJson();
+    }
+
+    /**
+     * Cancels the authorization.
+     *
+     * @throws ConnectorException        When the API replies with an error response
+     * @throws RequestException          When an error is encountered
+     * @throws \RuntimeException         On an unexpected API response
+     * @throws \RuntimeException         If the response content type is not JSON
+     * @throws \InvalidArgumentException If the JSON cannot be parsed
+     * @throws \LogicException           When Guzzle cannot populate the response
+     *
+     * @return self
+     */
+    public function cancelAuthorization()
+    {
+        $this->delete($this->getLocation())
+            ->status('204');
+            // ->contentType('application/json');
+            // TODO: We cannot check the Content-type here because of an inconsistency
```

Comment:
I just did it because our API declare the content-type header. And this header was declared as "application/json". So I posted a bug to the team, but also left this part of code.

If they decided to keep the header declaration - SDK must follow the rule.